### PR TITLE
feat(dragdrop): add custom drag and drop handler docs

### PR DESCRIPTION
**Pull request by peyerluk**
## Description

Add Documentation for custom Drag & Drop Handlers

### CODE REVIEW

File: reference-docs/editor-configuration/document-drag-drop.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Document Drag & Drop
+
+## Description
+
+The editor has some built in Drag & Drop implementations, to handle image, file and text drops onto a document. There is also an integration with the asset management system HuGO.
```
Comment (gabrielhase) [inline reference-docs/editor-configuration/document-drag-drop.md]: It's called huGO+ :) but I am not sure if you want to mention this here, sounds unrelated.

File: reference-docs/editor-configuration/document-drag-drop.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+It is also possible to add your own drag & drop handlers in the editor.
+
+Dropping objects over a dashboard is not covered in this document.
```
Comment (gabrielhase) [inline reference-docs/editor-configuration/document-drag-drop.md]: Rather, it's not covered by the API you implemented, isn't it?

File: reference-docs/editor-configuration/document-drag-drop.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## Add Custom Drag & Drop Handlers
+
+You can register your own drop handlers which will be called before the core drop handlers
+are called. In your plugin you can either call `next()` which will call the next drop handler, or you can call `end()` if you either handle the drop with this handler or wish to
+abort without handling the drop.
```
Comment (gabrielhase) [inline reference-docs/editor-configuration/document-drag-drop.md]: what happens in the browser if you abort a drop?

File: reference-docs/editor-configuration/document-drag-drop.md (hunk 1 above)
Comment (peyerluk) [reply reference-docs/editor-configuration/document-drag-drop.md]: I'll remove it.

File: reference-docs/editor-configuration/document-drag-drop.md (hunk 3 above)
Comment (peyerluk) [reply reference-docs/editor-configuration/document-drag-drop.md]: Nothing. It just ends without any user feedback.

**Pull request merged**